scripts/gee

```diff
@@ -2502,7 +2502,7 @@ function gee__commit() {
     fi
     # We always push upstream so that users have a backup in case they lose their
     # laptop:
-    _git push -u origin "${CURRENT_BRANCH}"
+    _git push --quiet -u origin "${CURRENT_BRANCH}"
```

Comment:
should there be a `_git_push` function? so we don't have the same option over and over? Your call.